Tests/PointFreeTests/SessionTests.swift

```diff
@@ -0,0 +1,82 @@
+import Models
+@testable import PointFree
+import PointFreeTestSupport
+import SnapshotTesting
+import XCTest
+
+final class SessionTests: TestCase {
+  override func setUp() {
+    super.setUp()
+//    record=true
+  }
+
+  func testEncodable() {
+    var session: Session
```

Comment:
im doing some local variable rewrite stuff so that I can keep the `_assertInlineSnapshot` nice and tidy. I had started by using new lines with `_assertInlineSnapshot` but then it doesn't find the `with` argument.